server/sonar-server/src/test/java/org/sonar/server/component/db/ComponentDaoTest.java

```diff
@@ -84,9 +88,9 @@ public void get_by_uuid() {
 
   @Test
   public void get_by_uuid_on_technical_project_copy() {
-    setupData("shared");
+    db.prepareDbUnit(getClass(), "shared.xml");
```

Comment:
forget it, just noticed that test methods with various initial dbunit setup are mixed in this class
my comment does not apply